server/src/modules/apps/ability/guard.ts

```diff
@@ -15,6 +15,10 @@ export class FeatureAbilityGuard extends AbilityGuard {
         return {
           resourceType: MODULES.APP,
         };
+      case APP_TYPES.FRONT_END || APP_TYPES.MODULE:
```

Comment:
case APP_TYPES.FRONT_END:
        return {
          resourceType: MODULES.APP,
        };
        
        
        why do we need to keep this?